packages/shared/dao/index.ts

```diff
@@ -51,6 +51,10 @@ export function isPeerHealthy(peerStatus: Record<string, HealthStatus>) {
   )
 }
 
+export function lambdasStatusUrl(domain: string) {
+  return `${domain}/lambdas/status`
```

Comment:
please use and check that the 3 services are "Healthy"
```suggestion
  return `${domain}/lambdas/health`
```